docs/docusaurus/docs/cloud/alerts/_examples/create_a_custom_action.py

```diff
@@ -27,13 +27,14 @@ class MyCustomAction(ValidationAction):
 
     # 2. Set the `type` attribute to a unique string that identifies the Action.
     # <snippet name="docs/docusaurus/docs/cloud/alerts/_examples/create_a_custom_action.py - set type">
-    type: Literal["my_custom_action"] = "my_custom_action"
+    type: Literal["my_custom_action"] = "my_custom_cloud_action"
```

Comment:
I'm surprised to see the two user-supplied values in this line no longer matching each other. This seems to be giving me an error

<img width="1126" height="63" alt="Screenshot 2025-11-20 at 1 00 58 PM" src="https://github.com/user-attachments/assets/ae953f04-5582-41f6-bb33-54426a016af5" />

Did this code work for you without any error?